app/models/alchemy/page.rb

```diff
@@ -114,10 +114,6 @@ class Page < ActiveRecord::Base
       if: -> { parent && parent.restricted? },
       unless: :systempage?
 
-    before_save :set_published_at,
-      if: -> { public? && published_at.nil? },
-      unless: :systempage?
```

Comment:
It actually gets called, when you hit the save button in the page settings overlay.